src/SignaturePad.Shared/SignaturePadCanvasView.cs

```diff
@@ -84,7 +90,11 @@ public NativeRect GetSignatureBounds (float padding = 5f)
 				return NativeRect.Empty;
 			}
 
+#if NET471
+			var size = this.RenderSize;
+#else
```

Comment:
This could also be an extension method: https://github.com/xamarin/SignaturePad/blob/master/src/SignaturePad.Shared/Extensions.cs

Reply:
+